lib/ssl/doc/src/ssl.xml

```diff
@@ -2175,6 +2173,22 @@ fun(srp, Username :: binary(), UserState :: term()) ->
           is useful.</p>
       </desc>
     </func>
+
+    <func>
+      <name since="OTP @OTP-18572@" name="signature_algs" arity="2" />
+      <fsummary>Returns a list of signature algorithms/schemes </fsummary>
+      <desc>
+        <p>Lists all possible signature algorithms corresponding to
+        <c>Description</c> that are available. The
+        <c>exclusive</c> option will exclusively list
+        algorithms/schemes for that protocol version, whereas the
+        default and all options lists the combined list to support the
+        range of protocols from (D)TLS-1.2, the first version to support
+        configuration of the signature algorithms, to <c>Version</c>. </p>
+	<note><p>Some TLS-1-3 scheme names overlap with TLS-1.2 tuple-pair-names
+	and then TLS-1.3 names will be used.</p></note>
+      </desc>
+    </func>
```

Comment:
Should we have an example? 
I don't think I understand the difference between all and default in this description. 
The default and all options in the description should be in <c> </c> tags.

Reply:
So I tried to address this, pleas read again!